static/js/telescope_availability_app.js

```diff
@@ -0,0 +1,23 @@
+import Vue from 'vue';
+import $ from 'jquery';
+import {siteCodeToName} from './utils.js';
```

Comment:
You can put these on the same line. {siteCodeToName, observatoryCodeToNumber} etc.